coastguard/clustersetnetworkpolicy.md

```diff
@@ -0,0 +1,272 @@
+# ClusterSetNetworkPolicy: An Approach to Multi-Cluster Network Policy
+
+Epic: [Multi cluster network policies](https://github.com/submariner-io/enhancements/issues/176)
+
+## Summary
+
+Kubernetes provides a mechanism for policing network access to pods - the [NetworkPolicy].
+The default mechanism allows specifying ingress and egress traffic rules for a set of peer pods (determined by a selector).
+A user could define network policies to allow traffic across the clusterset, by specifying peers using "ip blocks",
+but it would be impossible to use more abstract selectors to identify peers, as each cluster can only account for its own resources.
+
+We propose a new `ClusterSetNetworkPolicy` CRD for specifying multi cluster network policies.
```

Comment:
I hope we will have option to enable/disable this feature similar to "service-discovery"?